code/modules/mob/living/living.dm

```diff
@@ -975,6 +975,15 @@
 		visible_message("<span class='notice'>[user] butchers [src].</span>")
 		gib()
 
+/mob/living/proc/can_use(atom/movable/M, be_close=FALSE, no_tk=FALSE)
+	if(incapacitated())
```

Comment:
`incapacitated()` is basically deprecated. We want to check for `TRAIT_HANDS_BLOCKED` instead, which the mob also gets when they're stunned/weakened/restrained.
```suggestion
	if(HAS_TRAIT(src, TRAIT_HANDS_BLOCKED))
```